editor.planx.uk/src/ui/editor/SelectInput/SelectInput.tsx

```diff
@@ -11,6 +11,7 @@ export type Props = SelectProps & {
   children?: ReactNode;
   onChange?: SelectProps["onChange"];
   bordered?: boolean;
+  visuallyHiddenLabel?: boolean;
```

Comment:
This doesn't strike me a that great a pattern - a per previous comment we might want to rework this component a little more to optionally accept a label so the props would always be `PropsWithLabel | PropsWithoutLabel`. This would ensure that the component would always be a11y-compliant. There's some complexity here in that our labels are sometimes above, sometimes to the left, and sometimes visually hidden.

Going for the quick / pragmatic option here for now 👍